modules/Cockpit/module/assets.php

```diff
@@ -83,6 +83,10 @@
 
             $opts = ['mimetype' => $asset['mime']];
 
+            if (isset($meta[$idx]) && is_array($meta[$idx])) {
+                $meta = $meta[$idx];
```

Comment:
you're overriding the $meta array here. so it will only work for `$meta[0]` in the next iteration e.g. `$meta[1]` isn't available

Reply:
ups, my bad :( 
should be fixed now